src/Text/Pandoc/Readers/Docx.hs

```diff
@@ -356,10 +357,10 @@ extentToAttr _ = nullAttr
 blocksToInlinesWarn :: PandocMonad m => T.Text -> Blocks -> DocxContext m Inlines
 blocksToInlinesWarn cmtId blks = do
   let paraOrPlain :: Block -> Bool
-      paraOrPlain (Para _)  = True
-      paraOrPlain (Plain _) = True
-      paraOrPlain (Div _ _) = True
-      paraOrPlain _         = False
+      paraOrPlain (Para _)       = True
+      paraOrPlain (Plain _)      = True
+      paraOrPlain (Div _ nested) = (all paraOrPlain nested)
```

Comment:
This is the right approach.  But you don't need the `()` around `all paraOrPlain nested`.

Reply:
Adjusted the style accordingly